### ISSUE TITLE
ControlStructureSpacingSniff false positive

### ISSUE THREAD

**Issue by EvgenyOrekhov** [slevomat/coding-standard#487]
```php
<?php

declare(strict_types=1);

return;

```

Expected result: no warnings

Actual result:

> Expected 1 lines after "return", found -6.

Slevomat Coding Standard version: 4.8.0
PHP_CodeSniffer version: 3.3.1
PHP version: 7.2.8

**Comment by kukulich** [comment]
Thank you for your report.

Should be fixed in https://github.com/slevomat/coding-standard/commit/af1aeb948cc7abb67bbd318947966539e353bb86
